src/registrar/templates/django/admin/includes/detail_table_fieldset.html

```diff
@@ -0,0 +1,61 @@
+{% extends "admin/fieldset.html" %}
+{% load static url_helpers %}
+
+{% comment %}
+This is using a custom implementation fieldset.html (see admin/fieldset.html)
+{% endcomment %}
+{% block field_readonly %}
+    {% if field.field.name == "other_contacts" %}
+    <div class="readonly">
+        {% for contact in field.contents|split:", " %}
+            <a href="#" class="other-contact__{{forloop.counter}}">{{ contact }}</a>{% if not forloop.last %}, {% endif %}
+        {% endfor %}
+    </div>
+    {% elif field.field.name == "current_websites" %}
+    {% comment %}
+    The "website" model is essentially just a text field.
+    It is not useful to be redirected to the object definition,
+    rather it is more useful in this scenario to be redirected to the
+    actual website (as its just a plaintext string otherwise).
+
+    This ONLY applies to analysts. For superusers, its business as usual.
+    {% endcomment %}
+    {% for website in field.contents|split:", " %}
+        <a href="{{ website }}" class="padding-top-1 current-website__{{forloop.counter}}">{{ website }}</a>{% if not forloop.last %}, {% endif %}
+    {% endfor %}
+    {% else %}
+    <div class="readonly">{{ field.contents }}</div>
+    {% endif %}
+{% endblock field_readonly %}
+
+{% block after_help_text %}
+    {% if field.field.name == "creator" %}
+        {% include "django/admin/includes/contact_detail_table.html" with user=original.creator field_name="creator" %}
+    {% elif field.field.name == "submitter" %}
+        {% include "django/admin/includes/contact_detail_table.html" with user=original.submitter field_name="submitter" %}
+    {% elif field.field.name == "authorizing_official" %}
+        {% include "django/admin/includes/contact_detail_table.html" with user=original.authorizing_official field_name="authorizing_official" %}
+    {% elif field.field.name == "other_contacts" and original.other_contacts.all %}
+        <details class="margin-top-1 dja-detail-table">
+            <summary class="padding-1 padding-left-0 dja-details-summary">Details</summary>
```

Comment:
(Blocking): This is button that shows/hides content. It should be marked-up as a button, with a possible aria label - or better yet, dynamic inner HTM that toggles from Show details and Hide details